substratest/assets.py

```diff
@@ -385,6 +386,7 @@ class Meta:
 class ComputePlan(_Asset, _ComputePlanFutureMixin):
     compute_plan_id: str
     objective_key: str
+    status: str
```

Comment:
We could create a ComputePlan class, it would be more explicit and will be simpler to maintain.

If the status are exactly the same, we could simply do:

```python
class ComputePlanStatus(TupleStatus):
    pass
```